kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/security/ElytronUserGroupAdapter.java

```diff
@@ -36,14 +43,56 @@
         if (identityOptional.isPresent()) {
```

Comment:
to reduce indentation levels, you can refactor here and if not present, return empty list

Reply:
done

Reply:
done